client/components/home/Subscriptions.vue

```diff
@@ -1,22 +1,53 @@
 <script setup lang="ts">
+
+import BadgeButton from '@/components/buttons/BadgeButton.vue';
 import VideoEntry from '@/components/list/VideoEntry.vue';
+import { useSettingsStore } from '@/store/settings';
 
+const settingsStore = useSettingsStore();
 const { data: subscriptions, pending: subscriptionsLoading } = useGetUserSubscriptions({
-  limit: 4
+  limit: 20
+});
+
+const showMore = ref(false);
+const displayedSubscriptions = computed(() => {
+  if (subscriptions && subscriptions.value.videos) {
```

Comment:
```suggestion
  if (subscriptions.value && subscriptions.value.videos) {
```
Please add .value here ;)

Reply:
Otherwise it looks great!

Reply:
Done!